README.md

```diff
@@ -2,40 +2,77 @@
 
 The goal of this project is to provide a standard CSS implementation of the Spectrum design language for internal and 3rd party use on Adobe's platforms.
 
+1. [Show me a demo](#show-me-a-demo)
 1. [Where is the JavaScript?](#where-is-the-javascript)
 1. [Organization](#organization)
-2. [Using Spectrum CSS](#using-spectrum-css)
+1. [Using Spectrum CSS](#using-spectrum-css)
+    1. [CSS Custom Properties Strategy](#css-custom-properties-strategy)
     1. [Multi-stop Strategy](#multi-stop-strategy)
-    2. [Single-stop Strategy](#single-stop-strategy)
-3. [Building](#building)
-4. [Testing](#testing-with-the-visual-regression-tools)
-5. [Contributing](#contributing)
+    1. [Single-stop Strategy](#single-stop-strategy)
+1. [Building](#building)
+1. [Testing](#testing-with-the-visual-regression-tools)
+1. [Contributing](#contributing)
+
+## Show me a demo
+
+Check out [the documentation here](http://opensource.adobe.com/spectrum-css/), which shows every Spectrum CSS component and all of its variations on the same page.
```

Comment:
I figured just maybe we should link to our docs here in the readme, ya know?  ;)